requirements.txt

```diff
@@ -1 +1,2 @@
 pynwb
+PyYAML
```

Comment:
Oh, didn't even see that we had a requirements folder but weren't using it. Should we even keep this file, then? Or should we just update the setup to read from it?

Reply:
👍